pkg/descheduler/node/node.go

```diff
@@ -115,7 +115,6 @@ func PodFitsAnyNode(pod *v1.Pod, nodes []*v1.Node) bool {
 				klog.V(2).Infof("Pod %v can possibly be scheduled on %v", pod.Name, node.Name)
 				return true
 			}
-			return false
```

Comment:
You can drop entire `if ok` block, ok has to be true at this point.

Reply:
true, updated